src/harness/testcases/registration_testcase.py

```diff
@@ -147,3 +147,69 @@ def test_10_3_4_2_5_3(self):
     # Check registration response
     self.assertFalse('cbsdId' in response)
     self.assertEqual(response['response']['responseCode'], 103)
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_REG_26(self):
+    """Category Error in Array request (responseCode 202)
+
+    The response should be SUCCESS for the first CBSD,
+    FAILURE for the second, third, and fourth CBSDs.
+    """
+
+    device_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_4 = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+
+    # Set up unique FCC IDs
```

Comment:
Grouping is OK although we may not be able to maintain the scheme if we add more devices in the future.

I'm not opposed to changing the name (and if we did, I'd suggest using registration_* instead of device_* to match the other data files) although I worry a bit about trying to capture the parameters in the file name (why stop at category? why not also include location, EIRP, etc.?). It's not obvious to me what the best naming scheme is but I'm open to suggestions.